modules/shared/src/main/scala/org/tessellation/currency/schema/currency.scala

```diff
@@ -80,7 +90,8 @@ object currency {
     lastSnapshotHash: Hash,
     blocks: SortedSet[BlockAsActiveTip[CurrencyBlock]],
     tips: SnapshotTips,
-    info: CurrencySnapshotInfo
+    info: CurrencySnapshotInfo,
+    fee: SnapshotFee = SnapshotFee.MinValue
```

Comment:
You may need to register it in kryo.

Reply:
`TransactionFee` is not registered 🤔 

It is a `newtype` so at the very end it is just a `Long`

Reply:
fair point, it means that we definitely need to register `SnapshotFee`

Reply:
That depends on how are the newtypes treated during kryo serialization. If they are treated as base types then the registration may in fact be unnecessary.

Reply:
@TheMMaciek every time you see a newtype, think of it as an underlying type. the underlying type must be registered in kryo, because in fact newtype is just a "transparent wrapper". that being said, `Int` or `Long` are already registered, so it's all fine.
having:
```
@newtype
case class Foo(value: Bar)
```
you would have to register `Bar` type